### ISSUE TITLE
ImportError: DLL load failed:   from PyQt5 import QtCore, QtGui, QtWidgets

### ISSUE THREAD

**Issue by studyquant** [pyinstaller/pyinstaller#4420]
Dear pyinstaller Administrator，

```
(pyqt) E:\1. 工作内容\crypto>pyinstaller tradingwindows.py
151 INFO: PyInstaller: 3.5
152 INFO: Python: 3.6.2
157 INFO: Platform: Windows-10-10.0.17763-SP0
159 INFO: wrote E:\1. 工作内容\crypto\tradingwindows.spec
168 INFO: UPX is not available.
175 INFO: Extending PYTHONPATH with paths
['E:\\1. 工作内容\\crypto', 'E:\\1. 工作内容\\crypto']
176 INFO: checking Analysis
176 INFO: Building Analysis because Analysis-00.toc is non existent
177 INFO: Initializing module dependency graph...
181 INFO: Initializing module graph hooks...
185 INFO: Analyzing base_library.zip ...
4853 INFO: running Analysis Analysis-00.toc
4858 INFO: Adding Microsoft.Windows.Common-Controls to dependent assemblies of final executable
  required by c:\anaconda3-5.0.1\envs\pyqt\python.exe
5335 INFO: Caching module hooks...
5343 INFO: Analyzing E:\1. 工作内容\crypto\tradingwindows.py
5787 INFO: Processing pre-safe import module hook   urllib3.packages.six.moves
8306 INFO: Processing pre-find module path hook   distutils
11054 INFO: Processing pre-find module path hook   site
11055 INFO: site: retargeting to fake-dir 'c:\\anaconda3-5.0.1\\envs\\pyqt\\lib\\site-packages\\PyInstaller\\fake-modules'
12899 INFO: Processing pre-safe import module hook   setuptools.extern.six.moves
19870 INFO: Processing pre-safe import module hook   six.moves
23951 INFO: Loading module hooks...
23952 INFO: Loading module hook "hook-certifi.py"...
23956 INFO: Loading module hook "hook-distutils.py"...
23961 INFO: Loading module hook "hook-encodings.py"...
24183 INFO: Loading module hook "hook-lib2to3.py"...
24188 INFO: Loading module hook "hook-numpy.core.py"...
24407 INFO: Loading module hook "hook-numpy.py"...
24409 INFO: Loading module hook "hook-pandas.py"...
26334 INFO: Loading module hook "hook-pkg_resources.py"...
27088 INFO: Processing pre-safe import module hook   win32com
Traceback (most recent call last):
  File "<string>", line 2, in <module>
ModuleNotFoundError: No module named 'win32com'
27220 INFO: Processing pre-safe import module hook   win32com
Traceback (most recent call last):
  File "<string>", line 2, in <module>
ModuleNotFoundError: No module named 'win32com'
27343 INFO: Loading module hook "hook-pydoc.py"...
27345 INFO: Loading module hook "hook-PyQt5.py"...
27527 INFO: Loading module hook "hook-PyQt5.Qt.py"...
27542 INFO: Loading module hook "hook-PyQt5.QtCore.py"...
27544 INFO: Loading module hook "hook-PyQt5.QtGui.py"...
27547 INFO: Loading module hook "hook-PyQt5.QtHelp.py"...
27550 INFO: Loading module hook "hook-PyQt5.QtMultimedia.py"...
27551 INFO: Loading module hook "hook-PyQt5.QtNetwork.py"...
27552 INFO: Loading module hook "hook-PyQt5.QtOpenGL.py"...
27553 INFO: Loading module hook "hook-PyQt5.QtPrintSupport.py"...
27553 INFO: Loading module hook "hook-PyQt5.QtQml.py"...
27554 INFO: Loading module hook "hook-PyQt5.QtQuick.py"...
27555 INFO: Loading module hook "hook-PyQt5.QtQuickWidgets.py"...
27556 INFO: Loading module hook "hook-PyQt5.QtSensors.py"...
27557 INFO: Loading module hook "hook-PyQt5.QtSerialPort.py"...
27558 INFO: Loading module hook "hook-PyQt5.QtSql.py"...
27564 INFO: Loading module hook "hook-PyQt5.QtSvg.py"...
27566 INFO: Loading module hook "hook-PyQt5.QtTest.py"...
27568 INFO: Loading module hook "hook-PyQt5.QtWebEngineWidgets.py"...
27568 INFO: Loading module hook "hook-PyQt5.QtWidgets.py"...
27569 INFO: Loading module hook "hook-PyQt5.QtXml.py"...
27571 INFO: Loading module hook "hook-pytz.py"...
27781 INFO: Loading module hook "hook-setuptools.py"...
27866 INFO: Loading module hook "hook-sqlite3.py"...
28099 INFO: Loading module hook "hook-sysconfig.py"...
28100 INFO: Loading module hook "hook-xml.dom.domreg.py"...
28101 INFO: Loading module hook "hook-xml.py"...
28178 INFO: Looking for ctypes DLLs
28248 INFO: Analyzing run-time hooks ...
28258 INFO: Including run-time hook 'pyi_rth_certifi.py'
28262 INFO: Including run-time hook 'pyi_rth_multiprocessing.py'
28268 INFO: Including run-time hook 'pyi_rth_pkgres.py'
28270 INFO: Including run-time hook 'pyi_rth_pyqt5webengine.py'
28271 INFO: Including run-time hook 'pyi_rth_pyqt5.py'
28299 INFO: Looking for dynamic libraries
29226 WARNING: lib not found: Qt5XmlPatterns.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtXmlPatterns.pyd
29369 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtXmlPatterns.pyd
29534 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtXml.pyd
29676 WARNING: lib not found: Qt5Xml.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtXml.pyd
29834 WARNING: lib not found: Qt5Gui.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtWinExtras.pyd
30003 WARNING: lib not found: Qt5Widgets.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtWinExtras.pyd
30178 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtWinExtras.pyd
30342 WARNING: lib not found: Qt5WinExtras.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtWinExtras.pyd
30496 WARNING: lib not found: Qt5WebSockets.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtWebSockets.pyd
30629 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtWebSockets.pyd
30784 WARNING: lib not found: Qt5Network.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtWebSockets.pyd
30933 WARNING: lib not found: Qt5Gui.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtWebEngineWidgets.pyd
31076 WARNING: lib not found: Qt5WebEngineWidgets.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtWebEngineWidgets.pyd
31221 WARNING: lib not found: Qt5Widgets.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtWebEngineWidgets.pyd
31374 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtWebEngineWidgets.pyd
31534 WARNING: lib not found: Qt5WebEngineCore.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtWebEngineCore.pyd
31720 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtWebEngineCore.pyd
31904 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtWebEngine.pyd
32052 WARNING: lib not found: Qt5WebEngine.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtWebEngine.pyd
32198 WARNING: lib not found: Qt5WebChannel.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtWebChannel.pyd
32350 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtWebChannel.pyd
32484 WARNING: lib not found: Qt5Gui.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtTest.pyd
32617 WARNING: lib not found: Qt5Widgets.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtTest.pyd
32771 WARNING: lib not found: Qt5Test.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtTest.pyd
32922 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtTest.pyd
33076 WARNING: lib not found: Qt5Gui.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtSvg.pyd
33224 WARNING: lib not found: Qt5Widgets.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtSvg.pyd
33386 WARNING: lib not found: Qt5Svg.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtSvg.pyd
33544 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtSvg.pyd
33718 WARNING: lib not found: Qt5Gui.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtSql.pyd
33861 WARNING: lib not found: Qt5Sql.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtSql.pyd
34001 WARNING: lib not found: Qt5Widgets.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtSql.pyd
34142 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtSql.pyd
34288 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtSerialPort.pyd
34425 WARNING: lib not found: Qt5SerialPort.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtSerialPort.pyd
34557 WARNING: lib not found: Qt5Sensors.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtSensors.pyd
34708 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtSensors.pyd
34857 WARNING: lib not found: Qt5Gui.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtQuickWidgets.pyd
35025 WARNING: lib not found: Qt5QuickWidgets.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtQuickWidgets.pyd
35182 WARNING: lib not found: Qt5Widgets.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtQuickWidgets.pyd
35349 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtQuickWidgets.pyd
35518 WARNING: lib not found: Qt5Gui.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtQuick.pyd
35692 WARNING: lib not found: Qt5Qml.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtQuick.pyd
35860 WARNING: lib not found: Qt5Quick.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtQuick.pyd
36022 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtQuick.pyd
36229 WARNING: lib not found: Qt5Qml.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtQml.pyd
50869 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtQml.pyd
51049 WARNING: lib not found: Qt5Gui.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtPrintSupport.pyd
51224 WARNING: lib not found: Qt5PrintSupport.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtPrintSupport.pyd
51386 WARNING: lib not found: Qt5Widgets.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtPrintSupport.pyd
51542 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtPrintSupport.pyd
51698 WARNING: lib not found: Qt5Positioning.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtPositioning.pyd
51843 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtPositioning.pyd
51983 WARNING: lib not found: Qt5Gui.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtOpenGL.pyd
52129 WARNING: lib not found: Qt5Widgets.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtOpenGL.pyd
52319 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtOpenGL.pyd
52476 WARNING: lib not found: Qt5OpenGL.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtOpenGL.pyd
52653 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtNfc.pyd
52841 WARNING: lib not found: Qt5Nfc.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtNfc.pyd
53070 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtNetwork.pyd
53249 WARNING: lib not found: Qt5Network.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtNetwork.pyd
53429 WARNING: lib not found: Qt5Gui.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtMultimediaWidgets.pyd
53611 WARNING: lib not found: Qt5Widgets.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtMultimediaWidgets.pyd
53773 WARNING: lib not found: Qt5MultimediaWidgets.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtMultimediaWidgets.pyd
53914 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtMultimediaWidgets.pyd
54110 WARNING: lib not found: Qt5Multimedia.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtMultimedia.pyd
54263 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtMultimedia.pyd
54398 WARNING: lib not found: Qt5Positioning.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtLocation.pyd
54535 WARNING: lib not found: Qt5Location.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtLocation.pyd
54675 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtLocation.pyd
54851 WARNING: lib not found: Qt5Widgets.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtHelp.pyd
55062 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtHelp.pyd
55226 WARNING: lib not found: Qt5Help.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtHelp.pyd
55391 WARNING: lib not found: Qt5Gui.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtDesigner.pyd
55544 WARNING: lib not found: Qt5Widgets.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtDesigner.pyd
55701 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtDesigner.pyd
55850 WARNING: lib not found: Qt5Designer.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtDesigner.pyd
56009 WARNING: lib not found: Qt5DBus.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtDBus.pyd
56149 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtDBus.pyd
56315 WARNING: lib not found: Qt5Bluetooth.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtBluetooth.pyd
56455 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtBluetooth.pyd
56611 WARNING: lib not found: Qt5Gui.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtWidgets.pyd
56791 WARNING: lib not found: Qt5Widgets.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtWidgets.pyd
56991 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtWidgets.pyd
57170 WARNING: lib not found: Qt5Gui.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtGui.pyd
57334 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtGui.pyd
57495 WARNING: lib not found: Qt5Core.dll dependency of c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyQt5\QtCore.pyd
57718 INFO: Looking for eggs
57718 INFO: Using Python library c:\anaconda3-5.0.1\envs\pyqt\python36.dll
57722 INFO: Found binding redirects:
[]
57739 INFO: Warnings written to E:\1. 工作内容\crypto\build\tradingwindows\warn-tradingwindows.txt
58020 INFO: Graph cross-reference written to E:\1. 工作内容\crypto\build\tradingwindows\xref-tradingwindows.html
58087 INFO: checking PYZ
58088 INFO: Building PYZ because PYZ-00.toc is non existent
58092 INFO: Building PYZ (ZlibArchive) E:\1. 工作内容\crypto\build\tradingwindows\PYZ-00.pyz
60639 INFO: Building PYZ (ZlibArchive) E:\1. 工作内容\crypto\build\tradingwindows\PYZ-00.pyz completed successfully.
60683 INFO: checking PKG
60684 INFO: Building PKG because PKG-00.toc is non existent
60690 INFO: Building PKG (CArchive) PKG-00.pkg
60736 INFO: Building PKG (CArchive) PKG-00.pkg completed successfully.
60740 INFO: Bootloader c:\anaconda3-5.0.1\envs\pyqt\lib\site-packages\PyInstaller\bootloader\Windows-64bit\run.exe
60742 INFO: checking EXE
60743 INFO: Building EXE because EXE-00.toc is non existent
60744 INFO: Building EXE from EXE-00.toc
60747 INFO: Appending archive to EXE E:\1. 工作内容\crypto\build\tradingwindows\tradingwindows.exe
60764 INFO: Building EXE from EXE-00.toc completed successfully.
60770 INFO: checking COLLECT
60771 INFO: Building COLLECT because COLLECT-00.toc is non existent
60776 INFO: Building COLLECT COLLECT-00.toc
67406 INFO: Building COLLECT COLLECT-00.toc completed successfully.


(pyqt) E:\1. 工作内容\crypto\dist\tradingwindows>tradingwindows.exe
Traceback (most recent call last):
  File "tradingwindows.py", line 8, in <module>
ImportError: DLL load failed: 找不到指定的模块。
[302720] Failed to execute script tradingwindows



```

as you can see above code,  it cannot import line8 code,  while i checked code of line8,  it is

 **from PyQt5 import QtCore, QtGui, QtWidgets**

I do not understand , why it can not import PyQt5 modules?

Thank you very much .

**Comment by atvKumar** [comment]
Try copying over the missing dlls to your exe location.

**Comment by BoboTiG** [comment]
If you move the .exe into a folder that does not contain unicode characters at all, do you have more luck?

**Comment by therealrahulsahu** [comment]
Please resolve this bug as soon as possible.

**Comment by Legorooj** [comment]
Please try this with the latest version of PyInstaller, 3.6.

**Comment by bwoodsend** [comment]
Closing due to lack of response from the original author. Will re-open if you
(or anyone else who has this issue) gets back to us by commenting below.

**Comment by ObscureBrandon** [comment]
I'm facing a similar issue when importing specific modules from PyQt5 (from PyQt5 import QtCore, QtGui, QtWidgets). This is the error I get:

`ImportError: DLL load failed while importing QtCore: %1 is not a valid Win32 application.
[13668] Failed to execute script main`

Also, worth mentioning, when running the pyinstaller command to package my app, it doesn't include PyQt5 automatically even when I try importing it above using **import PyQt5**. I have to manually copy the folder from site-packages into my project folder. This happens with a couple of other modules as well. If there's any way to make it include them automatically that would be helpful as I am trying to package in 1 file. 

Running Pyinstaller on version 4.0

Thanks in advance.

**Comment by bwoodsend** [comment]
> it doesn't include PyQt5 automatically even when I try importing it above using import PyQt5.

Sounds like you're mixing you environments up. Are you sure the pyinstaller shim in your `PATH` points to a PyInstaller in the same site-packages dir? If not, raise a new issue.

**Comment by ObscureBrandon** [comment]
I am not quite sure what shim means, but I figured it would have something to do with the location my project is importing PyInstaller from. I tried running `pip show pyinstaller` inside my project dir and it points to the correct folder in site-packages.

`Name: pyinstaller
Version: 4.0
Summary: PyInstaller bundles a Python application and all its dependencies into a single package.
Home-page: http://www.pyinstaller.org/
Author: Hartmut Goebel, Giovanni Bajo, David Vierra, David Cortesi, Martin Zibricky
Author-email: None
License: GPLv2-or-later with a special exception which allows to use PyInstaller to build and distribute non-free programs (including commercial ones)
Location: c:\users\-----\appdata\roaming\python\python38\site-packages
Requires: pywin32-ctypes, pyinstaller-hooks-contrib, setuptools, pefile, altgraph
Required-by:`

**Comment by rokm** [comment]
What are the outputs of the following commands?
* `pip freeze`
* `where pyinstaller`

**Comment by ObscureBrandon** [comment]
pip freeze: `pyinstaller==4.0`
where pyinstaller: `C:\Users\----\AppData\Local\Programs\Python\Python38-32\Scripts\pyinstaller.exe`

**Comment by rokm** [comment]
> pip freeze: pyinstaller==4.0

Did `pip freeze` also list `PyQt5`?

**Comment by rokm** [comment]
And do you happen to have both 32-bit and 64-bit python 3.8 installed? 

Considering one path was:
`c:\users\-----\appdata\roaming\python\python38\site-packages`
and the other
`C:\Users\----\AppData\Local\Programs\Python\Python38-32\Scripts\pyinstaller.exe`
(note python38 vs. python38-32)

**Comment by ObscureBrandon** [comment]
Did pip freeze also list PyQt5?: Yes it does `PyQt5==5.15.1`

Your second point seems to be the issue, I have 2 installations of python apparently. The second (32-bit one) doesn't have the packages I was talking about that don't get included (including PyQt5 and some others) 

Not sure what I should do here though. Is there any way to make it point to the other python path? 

Nonetheless, great remark.

**Comment by rokm** [comment]
If `where python` points at the correct location, you should try running `python -m PyInstaller ...` instead of `pyinstaller ...`. 

Otherwise, you'll need to edit your `PATH` to give precedence to the correct python version.

**Comment by bwoodsend** [comment]
If you have absolutely no intention of ever using it you can just delete ` C:\Users\----\AppData\Local\Programs\Python\Python38-32\Scripts\pyinstaller.exe`.
